dockerfile/wildfly/common/Dockerfile

```diff
@@ -40,6 +61,22 @@ ENV CONFIG_DIR=$HOME/config
 ENV WILDFLY_PROPERTIES_FILE=$CONFIG_DIR/wildfly.properties
 ENV LAUNCH_JBOSS_IN_BACKGROUND=true
 
+ARG POM_GROUP_ID
+ARG POM_ARTIFACT_ID
+ARG POM_VERSION
+
+ARG TMP_DIR=$HOME/tmp
+ARG DOWNLOAD_DIR=$HOME/download
+
+COPY --from=download --chown=$SYSTEM_USER:$SYSTEM_USER_GROUP ${DOWNLOAD_DIR}/ ${TMP_DIR}/
+COPY --chown=$SYSTEM_USER:$SYSTEM_USER_GROUP dockerfile/script/wildfly-cli/module/logstash.cli $TMP_DIR/module/logstash.cli
+COPY --chown=$SYSTEM_USER:$SYSTEM_USER_GROUP \
+    dockerfile/script/wildfly-cli/embed-server/configure-logging.cli \
+    dockerfile/script/wildfly-cli/embed-server/configure-logging-file.cli \
+    dockerfile/script/wildfly-cli/embed-server/configure-logstash.cli \
```

Comment:
Ebben a formaban a scriptek egymas ellen dolgoznak. 4 loggert akarunk egyszerre tamogatni:
* CONSOLE
* CONSOLE_JSON
* FILE
* GELF_LOGGER (logstash)
A scriptek mind megvannak csak refaktorlani kellene oket. Vagy egybe rakni (nem ajanlott) vagy tematikusan elosztani - peldaul a file logger torolje is ki a "periodic-rotating-file-handler"-t ne a configure-logging.cli script (es meg vannak hasonlo furasagok)

Mind a 4 lehetoleg kellene hogy tudja:
* enable/disbale
* non/block

Amugymeg a kovetkezo sor "dockerfile/wildfly/common/embed-server.cli" pont ezt az elozo parancsokat tartalmazza

Reply:
block/nonblock: file es logstashnel ok, de a masik ketto hogyan tudna blockolni?

Reply:
"Amugymeg a kovetkezo sor "dockerfile/wildfly/common/embed-server.cli" pont ezt az elozo parancsokat tartalmazza"

ez copyzza, az embed-server.cli meg futtatja